<?xml version="1.0"?>
<launch>
    <arg name="uav0" default="iris0"/>
        <group ns="$(arg uav0)">
          <node name="state_manager" pkg="state_manager" type="state_manager_node" output="screen" clear_params="true">

            <rosparam command="load" file="$(find state_manager)/cfg/state_manager_config.yaml"/>

            <remap from="state" to="mavros/state"/>
            <remap from="control_state" to="control/state"/>
            <remap from="battery" to="mavros/battery"/>

            <remap from="armingclient" to="mavros/cmd/arming"/>
            <remap from="serviceclient" to="mavros/set_mode"/>
            <remap from="velocityclient" to="mavros/param/set"/>
          </node>
        </group>
</launch>
